cmd/atlas/internal/docker/docker_test.go

```diff
@@ -416,14 +433,20 @@ func TestImageURL(t *testing.T) {
 }
 
 func TestContainerURL(t *testing.T) {
-	c := &Container{cfg: Config{driver: "postgres"}, Passphrase: "pass", Port: "5432"}
+	c := &Container{
+		Config: Config{
+			driver: "postgres",
+			User:   url.UserPassword("postgres", "pass"),
+		},
+		Port: "5432",
+	}
 	u, err := c.URL()
 	require.NoError(t, err)
-	require.Equal(t, "postgres://postgres:pass@localhost:5432/?sslmode=disable", u.String())
+	require.Equal(t, "postgres://postgres:pass@localhost:5432?sslmode=disable", u.String())
```

Comment:
Why omitting the /?

Reply:
Is it important? because the database name is empty. So, I think no-path and `/` will have same effect.

Reply:
I don’t know tbh. If we change it, we need to test it on the different database drivers.